storage/HashTable.hpp

```diff
@@ -1429,6 +1492,11 @@ HashTablePutResult HashTable<ValueT, resizable, serializable, force_key_copy, al
                                        variable_size,
                                        (*functor)(*accessor),
                                        using_prealloc ? &prealloc_state : nullptr);
+            // Insert into bloom filter, if enabled.
+            if (has_build_side_bloom_filter_) {
+              this->build_bloom_filter_->insert(static_cast<const std::uint8_t *>(key.getDataPtr()),
```

Comment:
@hbdeshmukh Harshad, thread safety is being guaranteed here from inside the `build_bloom_filter_->insert()`, which implements an exclusive mutex lock to allow only one thread to modify the bloom filter.
For containment test, I guess we will not require a lock, since the assumption is that a bloom filter will never be used, unless it has been built.